ioctl/cmd/did/diddoc.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2020 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package did
+
+const (
+	// DIDPrefix is the prefix string
+	DIDPrefix = "did:io:"
+	// DIDAuthType is the authentication type
+	DIDAuthType = "Secp256k1VerificationKey2018"
+	// DIDOwner is the suffix string
+	DIDOwner = "#owner"
+)
+
+type (
+	authenticationStruct struct {
+		ID           string `json:"id,omitempty"`
+		Type         string `json:"type,omitempty"`
+		Controller   string `json:"controller,omitempty"`
+		PublicKeyHex string `json:"publicKeyHex,omitempty"`
```

Comment:
`Secp256k1 Koblitz public key values MUST either be encoded as a JWK or be encoded as the raw 33-byte public key value in Base58 Bitcoin format using the publicKeyBase58 property.`

@xinxin-crypto what encoding shall we use? `hex` or `base58`?

Reply:
We may use publicKeyHex (i.e., a hex encoded compressed secp256k1 public key) and this format can be converted to publicKeyJwk if needed (i.e., verify a JSON Web Signature).